docs/index.md

```diff
@@ -136,6 +161,15 @@ JWT_AUTH = {
     'JWT_RESPONSE_PAYLOAD_HANDLER':
     'rest_framework_jwt.utils.jwt_response_payload_handler',
 
+    'JWT_BLACKLIST_GET_HANDLER':
+    'rest_framework_jwt.utils.jwt_blacklist_get_handler',
+
+    'JWT_BLACKLIST_SET_HANDLER':
+    'rest_framework_jwt.utils.jwt_blacklist_set_handler',
+
+    'JWT_BLACKLIST_RESPONSE_HANDLER':
+    'rest_framework_jwt.utils.jwt_blacklist_response_handler',
+
```

Comment:
I think it does make sense to separate the views and utils into the subapp as well so its completely decoupled